website/docs/bestpractices/analytics/trino-best-practices.md

```diff
@@ -0,0 +1,642 @@
+---
+sidebar_position: 2
+sidebar_label: Trino on EKS Best Practices
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CollapsibleContent from '../../../src/components/CollapsibleContent';
+
+# Trino on EKS Best Practices
+[Apache Trino](https://trino.io/) deployment on [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks/) (EKS) delivers distributed query processing with cloud-native scalability. Organizations can optimize costs by selecting specific compute instances and storage solutions that match their workload requirements while they  combine the power of Trino with the scalability and flexibility of EKS using [Karpenter](https://karpenter.sh/).
+
+This guide provides prescriptive guidance for deploying Trino on EKS. It focuses on achieving high scalability and low cost through optimal configurations, effective resource management, and cost-saving strategies. We cover detailed configurations for popular file formats such as Hive and Iceberg. These configurations ensure seamless data access and optimize performance. Our goal is to help you set up a Trino deployment that is both efficient and cost-effective.
+
+We have a [deployment-ready blueprint](https://awslabs.github.io/data-on-eks/docs/blueprints/distributed-databases/trino) for deploying Trino on EKS, which incorporates the best practices discussed here.
+
+Refer to these best practices for the rational and further optimization/fine-tuning.
+
+<CollapsibleContent header={<h2><span>Apache Trino</span></h2>}>
+
+## Basics
```

Comment:
Do we need this section? It lists information about Trino but it doesn't really go deep into any of them. I would introduce Trino briefly in the first section above and refer to the official docs or our doc in the blueprint section.
A high level architecture diagram is very useful here too imo.

Reply:
Lets discuss on this one